jo-rdan/Free-Mentor
# #167921997 create users can signin endpoint

#### What does this PR do?
It creates an endpoint to sign in
#### Description of Task to be completed?
- Add validations to field entered by the user
- Write unit tests to test the endpoint 
#### How should this be manually tested?
- Clone this repo to your computer
- Start the server by typing **npm start**
- Open **POSTMAN** and use this endpoint **localhost:3000/api/v1/auth/signin** and enter email and password, if successful it should return a **200** status code
  if not it should return a relevant error
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#167921997](https://www.pivotaltracker.com/story/show/167921997)
